tests/test_templates_state_preparations.py

```diff
@@ -111,16 +112,9 @@ def test_error_num_qubits(self, basis_state, wires):
         """Tests that the correct error message is raised when the number
         of qubits doesn't match the number of wires."""
 
-        with pytest.raises(ValueError, match="Number of qubits must be equal to the number of wires"):
+        with pytest.raises(ValueError):
```

Comment:
Same question as @johannesjmeyer: why are the `match`es removed? These help ensure that the `ValueError` is coming from the part of the code we expect it to come from (could give false positives otherwise)

Reply:
Ok, convinced. I will at least a regular expression check that looks for part of the message.